axolotl-web/src/pages/About.vue

```diff
@@ -1,39 +1,41 @@
 <template>
-  <div class="about">
-    <img class="logo" src="/axolotl.png" alt="Axolotl logo">
-    <h1 class="title">Axolotl Beta {{ config.Version }}</h1>
-    <h2 v-translate class="subtitle">A cross-platform Signal client</h2>
-    <div class="description">
-      <translate>
-        This is a free and open source Signal client written in golang and
-        vuejs.
-      </translate>
-      <br>
-      <translate class="mr-1">
-        You can support the development of Axolotl either by filling
-      </translate>
-      <a
-        v-translate
-        href="https://github.com/nanu-c/axolotl/issues"
-        @click="openExtern($event, 'https://github.com/nanu-c/axolotl/issues')"
-      >issues at the bug tracker</a>.
-      <br>
-      <translate class="mr-1">or by becoming a</translate>
-      <a
-        href="https://www.patreon.com/bePatron?u=11219559"
-        @click="
-          openExtern($event, 'https://www.patreon.com/bePatron?u=11219559')
-        "
-      ><span v-translate>sponsor on patreon.</span></a><br>
-      <br>
-      <font-awesome-icon id="heart" icon="heart" />
-      <br>
-      <a
-        href="https://axolotl.chat"
-        @click="openExtern($event, 'https://axolotl.chat')"
-      >https://axolotl.chat</a>
+  <component :is="$route.meta.layout || 'div'">
+    <div class="about">
+      <img class="logo" src="/axolotl.png" alt="Axolotl logo">
+      <h1 class="title">Axolotl Beta {{ config.Version }}</h1>
+      <h2 v-translate class="subtitle">A cross-platform Signal client</h2>
+      <div class="description">
+        <translate>
+          This is a free and open source Signal client written in golang and
+          vuejs.
+        </translate>
+        <br>
+        <translate class="mr-1">
+          You can support the development of Axolotl either by filling
+        </translate>
+        <a
+          v-translate
+          href="https://github.com/nanu-c/axolotl/issues"
+          @click="openExtern($event, 'https://github.com/nanu-c/axolotl/issues')"
```

Comment:
Shouldn't it be `openExternal`?

Reply:
It's a call to the function below, so it's not so important how it's called. Now I changed the name.

Reply:
Ah ok! I thought, it was that Electron function

https://www.electronjs.org/docs/latest/api/shell#shellopenexternalurl-options

Reply:
My bad, I guess.